src/shared/models/currencies.model.ts

```diff
@@ -3,5 +3,3 @@ import type { LiteralUnion } from 'leather-styles/types';
 export type CryptoCurrencies = LiteralUnion<'BTC' | 'STX', string>;
 
 export type FiatCurrencies = 'USD' | string;
-
-export type Currencies = CryptoCurrencies | FiatCurrencies;
```

Comment:
Maybe the `currencies.model` needs to move to  `'@leather-wallet/models'` ?

Reply:
it's there already

Reply:
👍 I meant maybe we can remove this `models/currencies.model.ts` file from the extension and take `CryptoCurrencies` + `FiatCurrencies` from the package?

Reply:
ah, sure, I just wanted to do this in sep pr